raddb/mods-available/eap

```diff
@@ -250,6 +255,11 @@ eap {
 			#
 			ca_file = ${certdir}/rsa/ca.pem
 
+			#
+			#  verify_depth:: Maximum number of certificates we can traverse.
```

Comment:
"Maximum number of certificates we traverse when attempting to validate the certificate chain."

English is hard :(